diff --git a/models/mb8600/server.js b/models/mb8600/server.js
index 14ae10f..45d5781 100644
--- a/models/mb8600/server.js
+++ b/models/mb8600/server.js
@@ -111,7 +111,7 @@ function makeRequest() {
 			return;
 		}
 		
-		if (body.indexOf("parent.location='login.html'") > -1) {
+		if (body.indexOf("Motorola Cable Modem : Login") > -1) {
 			console.warn("Logging into modem...");
 			
 			var encodedPassword = encode(config.modem.password);
@@ -323,4 +323,4 @@ function sendBuffer() {
 }
 
 makeRequest();
-sendBuffer();
\ No newline at end of file
+sendBuffer();
